install/kubernetes/cilium/values.yaml

```diff
@@ -4232,3 +4232,46 @@ authentication:
 enableInternalTrafficPolicy: true
 # -- Enable LoadBalancer IP Address Management
 enableLBIPAM: true
+# -- Standalone DNS Proxy Configuration
+standaloneDnsProxy:
+  # -- Enable standalone DNS proxy(alpha feature)
+  enabled: false
+  # -- Roll out Standalone Dns proxy automatically when configmap is updated.
+  rollOutPods: false
+  # -- L7 proxy mode enables proxy for DNS traffic 
+  l7Proxy: true
+  # -- Standalone DNS proxy annotations
+  annotations: {}
+  # -- Standalone DNS proxy debug mode
+  debug: false
+  # -- Standalone DNS proxy port
+  ## -- Port on which the DNS proxy listens for DNS requests. It should match the port configured in cilium-agent's dnsProxy.proxyPort.
+  proxyPort: 10094
```

Comment:
Why has this to be a separate value? We can just use `dns.proxyPort` everywhere, right? And fail the Helm chart if it is not explicitly set.